### PR TITLE
More resprites (Aug 2023)

### PR DESCRIPTION
**Pull request by nauticall**
* Resprited the following things: IV drips, gas tanks (the handheld ones), the O2/CO2 jetpacks, laptops, medical wristbounds, chemical dispenser cartridges, and cash (now actual credit chips).
* Reworked the way IV drips fall over when tipped to be less sprite-tedious, and removed being able to trip on them. They still fall over.
* Various backend code changes to make the modernized objects more up-to-speed with their contemporaries (i.e. chemical dispensers now have physical lids).

Basically, a sizeable amount of resprites to legacy things, as well as a bit of code mixed in between to make the changes fit. Hopefully doesn't break royally. Enjoy!

image

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
we already have chargecards, so there may be some confusion on the cash resprite.

**Comment by nauticall** [comment]
image
no idea why icondiffbot is freaking out over some of the IV sprites, they look fine on my end

**Comment by SleepyGemmy** [comment]
we already have chargecards, so there may be some confusion on the cash resprite.

**Comment by nauticall** [comment]
that was painful

**Review by NonQueueingMatt — APPROVED** [review]

**Review by DreamySkrell — APPROVED** [review]
